Makefile

```diff
@@ -24,5 +24,11 @@ omnibus: install
 distclean:
 	@rm -rf _build
 
-shell:
-	if [ -z $$(docker images -q $(DOCKER_IMAGE_TAG)) ]; then docker build --tag $(DOCKER_IMAGE_TAG) .; fi; docker run --volume $(PWD):/root --interactive --tty $(DOCKER_IMAGE_TAG)
+update-image:
+	@docker build --tag $(DOCKER_IMAGE_TAG) .
```

Comment:
Yes, but this only adds about a second (at least in local testing, may be more on a docker mac setup?)